src/quickActions/Funding_Request__c.Create_Bulk_Disbursements.quickAction

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
+    <height>500</height>
+    <label>Create Bulk Disbursements</label>
```

Comment:
```suggestion
    <label>Create Disbursements</label>
```

Reply:
Discussed as a change at the SFDO Sprint in Detroit. Was group's recommendation.